GraphNeuralNetworks/docs/src/tutorials/temporal_graph_classification.md

```diff
@@ -0,0 +1,196 @@
+```@meta
+EditURL = "../../src_tutorials/introductory_tutorials/temporal_graph_classification.jl"
+```
+
+# Temporal Graph classification with GraphNeuralNetworks.jl
+
+In this tutorial, we will learn how to extend the graph classification task to the case of temporal graphs, i.e., graphs whose topology and features are time-varying.
+
+We will design and train a simple temporal graph neural network architecture to classify subjects' gender (female or male) using the temporal graphs extracted from their brain fMRI scan signals. Given the large amount of data, we will implement the training so that it can also run on the GPU.
+
+## Import
+
+We start by importing the necessary libraries. We use `GraphNeuralNetworks.jl`, `Flux.jl` and `MLDatasets.jl`, among others.
+
+````julia
+using Flux
+using GraphNeuralNetworks
+using Statistics, Random
+using LinearAlgebra
+using MLDatasets: TemporalBrains
+using CUDA # comment out if you don't have a CUDA GPU
+
+ENV["DATADEPS_ALWAYS_ACCEPT"] = "true"  # don't ask for dataset download confirmation
+Random.seed!(17); # for reproducibility
+````
+
+## Dataset: TemporalBrains
+The TemporalBrains dataset contains a collection of functional brain connectivity networks from 1000 subjects obtained from resting-state functional MRI data from the [Human Connectome Project (HCP)](https://www.humanconnectome.org/study/hcp-young-adult/document/extensively-processed-fmri-data-documentation).
+Functional connectivity is defined as the temporal dependence of neuronal activation patterns of anatomically separated brain regions.
+
+The graph nodes represent brain regions and their number is fixed at 102 for each of the 27 snapshots, while the edges, representing functional connectivity, change over time.
+For each snapshot, the feature of a node represents the average activation of the node during that snapshot.
+Each temporal graph has a label representing gender ('M' for male and 'F' for female) and age group (22-25, 26-30, 31-35, and 36+).
+The network's edge weights are binarized, and the threshold is set to 0.6 by default.
+
+````julia
+brain_dataset = TemporalBrains()
```

Comment:
not necessarily blocking this PR, but we should add an option to the `TemporalBrains` constructor to download a subset of the dataset, so that we don't need to download gigabytes of data to run a tutorial.